### ISSUE TITLE
Customize the parameter used to map intercom users

### ISSUE THREAD

**Issue by Esya** [ForestAdmin/forest-express#145]
Currently, the integration uses a field called "email" on the model provided by the `mapping` parameter to retrieve the associated user in Intercom : 

```
./attributes-getter.js:        return intercom.users.find({ email: customer.email });
./conversations-getter.js:            email: customer.email,
```

This needs to be a parameter for when the email field has another name (like us), or we could just rely on the ID which is also stored on intercom.

**Comment by arnaudbesnier** [comment]
Hi @Esya,

You're right. Other integrations can be configured this way.
We need to improve this but I can't give you any ETA.

**Comment by cice** [comment]
👍

**Comment by arnaudbesnier** [comment]
Hi @Esya, @cice this improvement has been released in [v7.1.0](https://github.com/ForestAdmin/forest-express/releases/tag/v7.1.0) of the package.

You can now upgrade your agent and configure you Intercom integration this way:
https://docs.forestadmin.com/documentation/reference-guide/integrations#intercom

🌲🌲🌲
